composer.json

```diff
@@ -6,7 +6,11 @@
         "phpunit/phpunit": "~5.6",
         "goetas-webservices/xsd2php":"^0.2",
         "symfony/console": "v3.2.9",
-        "friendsofphp/php-cs-fixer": "^2.5"
+        "friendsofphp/php-cs-fixer": "^2.5",
+        "symfony/finder": "3.2.13",
```

Comment:
Wouldn't it be enough to set the versions of the symfony dependencies to `3.3.6`?

Reply:
Props for finding a newer version that supports the relevant PHP version!

Reply:
On second thought, transitive dependencies make a mess with more recent versions, so I chose to stick with the ones we know to work, for compatibility's sake.